docs/source/canfar.md

```diff
@@ -0,0 +1,94 @@
+# Running `shapepipe` on the canfar science portal
```

Comment:
General instructions hotwo run ShapePipe on the canfar science portal run.
Test, submit, monitor, and delete jobs.